www/static/favicon.svg

```diff
@@ -1,39 +1,87 @@
+    <use href="#oif-segment" fill="var(--blue)"   transform="rotate(216)" />
+    <use href="#oif-segment" fill="var(--yellow)" transform="rotate(288)" />
   </g>
 
-  <!-- https://www.svgrepo.com/svg/99963/rocket-pointing-to-upper-left-direction -->
-  <g id="rocket" transform="scale(-0.6,0.6) translate(-600,70)">
-    <path d="M208.195,373.272l24.652,88.625c0.287,1.062,0.908,1.998,1.75,2.707l75.802,63.035c0.985,0.812,2.2,1.244,3.443,1.244 c0.535,0,1.052-0.078,1.577-0.24c1.74-0.535,3.089-1.902,3.596-3.643l26.039-89.486c18.972,0.766,35.716-3.576,49.495-13.043 l34.502,28.344c2.189,1.787,5.374,1.605,7.334-0.449l25.618-26.957c0.994-1.043,1.521-2.449,1.473-3.902 c-0.048-1.443-0.68-2.811-1.75-3.787l-34.157-31.326c2.916-7.955,8.147-27.779,2.601-57.939l87.784-32.053 c1.712-0.622,2.983-2.066,3.395-3.835c0.421-1.769-0.096-3.624-1.349-4.944l-67.97-71.413c-0.794-0.832-1.817-1.387-2.955-1.597 l-89.533-16.007c-16.85-19.479-36.453-39.188-58.284-58.618C175.548,26.125,77.284,9.428,29.902,9.428 c-15.596,0-24.193,1.894-24.556,1.97c-2.133,0.478-3.758,2.199-4.131,4.351c-0.956,5.565-21.401,138.178,151.174,310.753 C152.618,326.721,175.749,349.145,208.195,373.272z M216.352,276.806c-30.026,0-54.363-24.337-54.363-54.363 c0-30.026,24.336-54.362,54.363-54.362s54.363,24.336,54.363,54.362C270.714,252.469,246.378,276.806,216.352,276.806z"/>
-    <path d="M420.97,471.278c-1.166-1.146-2.859-1.605-4.456-1.215c-1.597,0.393-2.878,1.568-3.395,3.117l-12.364,36.73 c-0.593,1.76-0.115,3.701,1.224,4.982l58.638,56.045c0.898,0.852,2.075,1.33,3.309,1.33l0,0h49.429l0,0 c1.95,0,3.7-1.186,4.437-2.984c0.727-1.807,0.287-3.873-1.109-5.23L420.97,471.278z"/>
-    <path d="M486.454,441.329c-0.908-0.947-2.161-2.936-3.462-2.936c-0.038,0-0.076,0-0.124,0c-1.349,0-2.61,2.084-3.49,3.107 l-21.258,26.049c-1.625,1.883-1.539,4.695,0.201,6.473l94.745,96.82c0.898,0.918,2.133,1.436,3.414,1.436l0,0h15.663h4.781 c2.64,0,4.781-2.143,4.781-4.781v-22.893c0-1.225-0.479-3.137-1.319-4.027L486.454,441.329z"/>
-    <path d="M580.262,453.11l-68.458-66.729c-1.587-1.559-4.045-1.797-5.89-0.611l-28.028,17.748c-1.233,0.785-2.036,2.074-2.189,3.52 c-0.153,1.443,0.354,2.887,1.387,3.91l96.485,95.701c0.918,0.91,2.142,1.387,3.376,1.387c0.611,0,1.243-0.123,1.836-0.373 c1.769-0.736,2.926-2.477,2.926-4.408v-46.73C581.706,455.243,581.18,453.999,580.262,453.11z"/>
+  <g transform="rotate(36)">
+    <!-- Rocket -->
+    <path class="rocket"
+      d="M0-141
+       A820,820 0 0 1 72,103
+       A270,270 0 0 0 0,16.5
+       A270,270 0 0 0 -72,103
+       A820,820 0 0 1 0-141"/>
+
+    <!-- Star -->
+    <path transform="translate(0-36) scale(1.5)" class="star"
+      d="M0-16 3.652-4.76h11.817l-9.56 6.944 3.65 11.24L0 6.474l-9.561 6.947
+          3.652-11.24-9.56-6.943h11.817L0-16z"/>
+
+    <!-- Big flame -->
+    <use href="#drop" class="gas"  transform="translate(0 40) scale(1.3 1.4)" />
+
+    <!-- Little flame -->
+    <use href="#drop" class="star" transform="translate(0 39) scale(0.8 0.7)" />
```

Comment:
I think the empty lines and possibly the comments are unnecessary, but I’ll trust your judgement on this.